src/components/HeaderWithLanguage/HeaderInformation.jsx

```diff
@@ -106,7 +106,7 @@ const HeaderInformation = ({
   createMessage,
   getArticle,
   history,
-  hasMultipleTaxonomyEntries,
```

Comment:
hvorfor har denne fått nytt navn? :-) Er det ikke en feature for om artikkelen har flere oppføringer i taksonomi også eller er det ikke relevant her? 
https://editorial-frontend-pr-805.ndla.sh/subject-matter/learning-resource/5875/edit/en

Reply:
Eg endra den fordi eg sender inn array av paths i stedet for en boolean. Sjekken på om den har fleire entries er flytta til HeaderStatusInformation linje 78. Her sjekker eg om paths > 2 fordi eg legger på /article-url som fallback dersom det ikkje finnes taksonomi-urler.